sdks/client-go/README.md

```diff
@@ -121,6 +121,18 @@ The client SDK provides the ability to generate analytics events with the `LogAn
 Currently the SDK only supports a logging analytics collector, which is configured by default whenever you use the SDK. It logs events to the console at DEBUG level.
 
 
```

Comment:
heya, was wondering if we could provide a Go server example to demo how this can be used?

Reply:
What are you thinking... some kind of HelloWorld API perhaps?

Reply:
I think i can do this, but i'd want to make a separate PR for it